### PR TITLE
Update for launch

### PR DESCRIPTION
**Pull request by mermop**
- Update copy for launch
- Standardise on Kiwi Ruby rather than KiwiRuby
- Add Google Analytics
- Add OpenGraph meta tags

I keep seeing a weird flicker when the page loads - @tobydallder @libbyberrie do you know anything about why this is occurring and if there's a way to fix it? My guess is it's something to do with metaquery, which I haven't used before, so :woman_shrugging: 

~I feel like the copy is a bit dry and needs more emoji, open to suggestions for emoji placement~ 
Added emoji

~@libbyberrie @tobydallder has this been browser tested? Happy to do it if not~
All good on Edge+ (older devices don't believe in 🤔 and IE doesn't believe in Young Serif, I'm fine with both of those compromises)

### Screenshots
All on chrome/mac os x
<details>
<summary>Desktop width</summary>

screencapture-localhost-4567-1493201384393
</details>

<details>
<summary>iPad width</summary>

screencapture-localhost-4567-1493201407157
</details>

<details>
<summary>iPhone 5 width</summary>

screencapture-localhost-4567-1493201393960
</details>

### CODE REVIEW

File: source/assets/public/index.css [hunk 1]
```diff
@@ -130,12 +130,9 @@ body {
     line-height: 4rem;
     text-align: left;
   }
-    .breakpoint-phone .hero__slogan span {
-      font-size: 3rem;
-      line-height: 4rem;
-    }
-  .breakpoint-phone .hero__date {
-    font-size: .1rem;
```
Comment (mermop) [inline source/assets/public/index.css]: Why was this here? Happy to add it back in, it just didn't seem to be used

File: source/index.html.slim [hunk 2]
```diff
@@ -23,42 +23,35 @@ title: Home
             h1.hero__slogan
               span.hero__hidden KiwiRuby
               br
-              | The ruby conference you
```
Comment (mermop) [inline source/index.html.slim]: Couldn't think of a better tag line but this wasn't quite right so leaving this out for the moment

File: source/index.html.slim [hunk 3]
```diff
@@ -23,42 +23,35 @@ title: Home
...
           p
-            ' Tautem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut voluptates repudiandae molestiae non. Itaque earum rerum hic tenetur a sapiente delectus.
+            ' KiwiRuby is a new Ruby conference being held in Wellington, New Zealand.
           p
-            ' Itaque earum rerum hic tenetur a sapiente delectus. There are more details to come. Follow us on
-            a> href="https://twitter.com/RubyConfNZ" Twitter
-            ' to stay updated!
-        .body-inner-wrapper__section
-          h2 When? Where?
+            ' With a mixture of local and international speakers, there will be one day of workshops and one single-track day of talks about topics that interest, excite and delight Rubyists.
```
Comment (mermop) [inline source/index.html.slim]: @parndt can you read over this copy?

**Comment by mermop** [comment]
@libbyberrie - should we just merge `master` into `gh-pages` when it's ready to go, or is `middleman-deploy` set up?

File: source/index.html.slim [hunk 4]
```diff
@@ -23,42 +23,35 @@ title: Home
...
+                ' November 2-3 2017
+              p.hero__date
+                ' Wellington, NZ
     section
       .body-inner-wrapper
         .body-inner-wrapper__section
           h2 Tell me more
           p
-            ' Tautem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut voluptates repudiandae molestiae non. Itaque earum rerum hic tenetur a sapiente delectus.
+            ' KiwiRuby is a new Ruby conference being held in Wellington, New Zealand.
```
Comment (parndt) [inline source/index.html.slim]: Here we say `KiwiRuby` but below we say `Kiwi Ruby`

File: source/index.html.slim (hunk 2 above)
Comment (parndt) [reply source/index.html.slim]: "Kiwi As" 😉 :trollface:

File: source/index.html.slim (hunk 3 above)
Comment (parndt) [reply source/index.html.slim]: Needs an oxford comma - "interest, excite, and delight Rubyists"

I wonder if it might appeal to more than just Rubyists?

Comment (mermop) [reply source/index.html.slim]: Can you think of a better way to phrase it? I had something like "Rubyists, Ruby-adjacent computer friends, and the Ruby-curious" but there were already quite a lot of commas in the sentence.. maybe this should be two sentences?

> With a mixture of local and international speakers, there will be one day of workshops and one single-track day of talks. We'll cover topics that interest, excite, and delight Rubyists and the Ruby-curious of all levels. 

How else should we describe who we're aiming at @parndt ?

File: source/index.html.slim (hunk 4 above)
Comment (mermop) [reply source/index.html.slim]: I think I've fixed this one

File: source/index.html.slim (hunk 3 above)
Comment (parndt) [reply source/index.html.slim]: I guess mainly I wanted to appeal to web developers and programmers in general who may only have a passing interest in Ruby

**Comment by mermop** [comment]
I deployed the site because a) insomnia and b) getting the OpenGraph tags working - I was thinking that I'd revert to the placeholder until Libby or Toby had a look through but I've just remembered that Icelab doesn't work on Fridays so they definitely won't look at this tomorrow.. I might undo some potentially contentious css changes and stick to the content

File: source/assets/public/index.css (hunk 1 above)
Comment (libbyberrie) [reply source/assets/public/index.css]: Yeah the `.breakpoint-phone .hero__slogan span` can be removed now. When we were using Young serif for the slogan there were weird kerning issues with the apostrophe, so i needed to surround it in a span. That condition was to make sure the apostrophe scaled down alongside the rest of the text. 


I'm not sure why the `hero__date` was so small in the small version. that can be removed no problem. :)

File: source/index.html.slim (hunk 2 above)
Comment (libbyberrie) [reply source/index.html.slim]: 100% ruby (if you want ultimate cheese)

**Comment by parndt** [comment]
@mermop I think we can safely merge this to master/gh-pages now? 😄

**Comment by libbyberrie** [comment]
If you give me a little time I'll see what I can do about the "glitchiness" on load - responsive layout changes are taking a while to load in.

**Comment by libbyberrie** [comment]
there are also a couple bits of input and CSS changes i may suggest ;)

File: source/assets/public/index.css [hunk 5]
```diff
@@ -216,7 +219,7 @@ body {
   }
   .breakpoint-desktop-up .body-inner-wrapper h2 {
     font-size: 3rem;
-    letter-spacing: .4rem;
+    letter-spacing: .75rem;
```
Comment (libbyberrie) [inline source/assets/public/index.css]: the .75rem was initially to make it look closer to the sketch layout but I'm happy to keep it at .4rem :)

**Comment by libbyberrie** [comment]
As the date is centered in the phone and tablet breakpoints, the left alignment of the birb logo is looking a little off at tablet resolution: 
image
Would you be OK if i centred the logo at that breakpoint?

**Comment by mermop** [comment]
Go ahead @libbyberrie

**Comment by libbyberrie** [comment]
We're getting some very weird capping issues when testing on other browsers. This is what was found on a colleague's browser:
losing-branding-on-wide-screens_360

This seems to be a browser end issue, as it is failing to display 
`    -webkit-font-feature-settings: "c2sc";
      font-feature-settings: "c2sc";`
on his browser, but it's showing up fine on my browser as well as being present in `/assets/public.css`. One way we can work around that is to just change the text in the .twig file as these details are the least likely to change over time. We could either just lowercase all the text and let allcaps do the rest, or put it in all caps and style the surrounding div (or surround it in a span) in order to reduce the letter height. 

Would probably be better to do the latter, as the numbers in the above image are also being affected

**Comment by libbyberrie** [comment]
I also think that having young serif as the 'Kiwi Ruby' heading at larger resolutions looks a little too much like it has defaulted to reset CSS (at least in my opinion). I could change it to Oleo Swash caps (what is used in the OG logotype) however that font doesn't support macrons. Would that be OK or should I just leave it as?

**Comment by libbyberrie** [comment]
Fixed the flickering. Pull down the branch and let me know if you're still experiencing it. :)

**Comment by mermop** [comment]
@libbyberrie - which browser was that with the problem above?

Happy to merge and fix those problems separately (since this is already deployed)
